src/api/v1/__tests__/auth_flow.test.integration.ts

```diff
@@ -43,75 +43,6 @@ describe('Auth flow', () => {
 
       expect(resAccess.statusCode).toBe(200);
     });
-
-    test('CB login from visitor :: escalate session :: de-escalate session', async () => {
```

Comment:
The `describe` block here is still called `Escalation`. Should probs be changed.